addons/website_sale/models/product.py

```diff
@@ -200,7 +200,7 @@ def _website_price(self):
         ret = self.env.user.has_group('sale.group_show_price_subtotal') and 'total_excluded' or 'total_included'
 
         for p, p2 in zip(self, self2):
-            taxes = partner.property_account_position_id.map_tax(p.taxes_id).filtered(lambda x: x.company_id == company_id)
+            taxes = partner.property_account_position_id.map_tax(p.sudo().taxes_id.filtered(lambda x: x.company_id == company_id))
```

Comment:
Why `sudo()`? Please don't mix multi-company issues. That should be treated in another issue/PR. Any way, I don't see here the problem without sudo, unless the user of `p` is not admin nor public user.